app/view_models/auction_show_view_model.rb

```diff
@@ -76,7 +76,8 @@ def bid_status_partial
     if !available? ||
        user_not_vendor? ||
        reverse_auction_available_user_is_winner? ||
-       sealed_bid_auction_user_is_bidder?
+       sealed_bid_auction_user_is_bidder? ||
+       current_user.sam_status != 'sam_accepted'
```

Comment:
this conditional is getting super ugly -- will make it better once we have a status for every type of user